### DISCUSSION TITLE
Create Quasar-like documentation

### DISCUSSION THREAD

**Discussion by Davdup** [quasarframework/quasar#9593] [General - Components / Directives / etc] [answered]
Hi,
I would really like to create the same kind of documentation pages as the ones on the Quasar-dev documentation. I downloaded the source files from here, had a look at the various files, but I'm a little lost... is there a quick way to render these .md files and get the navigation menu on the right? Are there specific extensions to add in order to use the .md formatting? what about the rouer.js file in this case??
Thanks for your help!

David

**Comment by hawkeye64** [comment, marked as answer]
@Davdup The magic happens in the quasar.conf.js file. Look for where it is using a custom md (markdown) loader. That loader you will find in the project.
